base/src/main/java/com/smartdevicelink/managers/lifecycle/RpcConverter.java

```diff
@@ -157,14 +160,11 @@ public static RPCMessage convertTableToRpc(Hashtable<String,Object> rpcHashTable
                     if(rpcConstructor != null){
                         return (RPCMessage)rpcConstructor.newInstance(rpcHashTable);
                     }
+                } else {
+                    DebugTool.logError(TAG + " Java class cannot be found for " + rpcClassName.toString());
                 }
-            } catch (ClassNotFoundException e) {
-            } catch (NoSuchMethodException e) {
-            } catch (IllegalAccessException e) {
-            } catch (InstantiationException e) {
-            } catch (InvocationTargetException e) {
-            } catch (ClassCastException e){
-
+            } catch (Exception e) {
+                DebugTool.logError(e.getMessage(), e);
```

Comment:
Change the first message to a more specific message `RPCConverter was unable to process RPC` or similar.